specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/privateLinkResources.json

```diff
@@ -0,0 +1,130 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SynapseManagementClient",
+    "description": "Azure Synapse Analytics APIs"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/privateLinkResources": {
+      "get": {
+        "operationId": "PrivateLinkResources_List",
```

Comment:
>List [](start = 45, length = 4)

there should also be an operation to get an individual privateLinkResource by name

Reply:
> > List [](start = 45, length = 4)
> 
> there should also be an operation to get an individual privateLinkResource by name

Backend doesn't provide such api, I think that list all privateLinkResource is ok.

Reply:
A collection GET that exposes entities with resource IDs needs individual GETs for those resources as well.  Some RPs have gotten away without doing this but the majority have done it correctly (search through the repo for examples).

Reply:
> A collection GET that exposes entities with resource IDs needs individual GETs for those resources as well. Some RPs have gotten away without doing this but the majority have done it correctly (search through the repo for examples).

Hi @pilor , Thanks for your comment. I have checked with backend team. Now our service doesn't need to support individual GETs. I think that this is not a block issue right?  Can we ignore this？

Reply:
No, this violates the RPC and must be implemented

Reply:
> No, this violates the RPC and must be implemented. Refer to cosmos DBs implementation for a good example

Added